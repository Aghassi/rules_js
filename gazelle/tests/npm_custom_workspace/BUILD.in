# gazelle:ts_npm_package_json my_package.json
# gazelle:ts_npm_package_workspace my_npm