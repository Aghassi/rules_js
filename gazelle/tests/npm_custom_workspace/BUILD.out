load("@npm//@bazel/typescript:index.bzl", "ts_project")

# gazelle:ts_npm_package_json my_package.json
# gazelle:ts_npm_package_workspace my_npm

ts_project(
    name = "npm_custom_workspace",
    srcs = ["main.ts"],
    deps = ["@my_npm//jquery"],
)